safety-architecture/tools/callgraph-tool/README.md

```diff
@@ -33,17 +33,12 @@ git clone https://github.com/elisa-tech/workgroups.git
 CG_DIR=$(pwd)/workgroups/safety-architecture/tools/callgraph-tool
 ```
 
-Install the following requirements:
+Install prerequisites with:
 ```
-sudo apt install python3 python3-pip build-essential cmake \
```

Comment:
Let's keep the README installation instructions as they were before. 

We can include the "install-apt-requirements" and "activate-venv" targets to the main makefile to automate your workflow, but at least I would still like the instructions to guide the workflow as it was before this change